rest/serializers.py

```diff
@@ -289,3 +290,19 @@ class LearningResourceExportTaskSerializer(Serializer):
     id = CharField()
     status = CharField()
     url = CharField()
+
+
+class RepositorySearchSerializer(Serializer):
+    """Serializer for search metadata."""
+    text = CharField()
```

Comment:
This matches up with `text` in `LearningResourceIndex`. It's basically a concatenation of title, description, and content, see `prepare_text` in `LearningResourceIndex`